packages/@local/eslint-config/index.js

```diff
@@ -250,184 +239,23 @@ module.exports = {
         "global-require": "off",
       },
     },
+    // {
+    //   files: ["packages/block-template-*/**/dev.tsx"],
+    //   rules: {
+    //     "import/no-extraneous-dependencies": [
+    //       "error",
+    //       { devDependencies: true },
+    //     ],
+    //   },
+    // },
+    // {
+    //   files: ["packages/block-template-custom-element/src/app.ts"],
+    //   rules: {
+    //     "no-console": "off",
+    //   },
+    // },
```

Comment:
TODO: figure out what to do with ESLint in block templates. We can't use our local rules there, unless we want to magically remove `.eslintrc.cjs` file and mentions of `@local/eslint-config` and `@local/eslint-config` in `package.json`.